### ISSUE TITLE
CI4 Composer postInstall Error

### ISSUE THREAD

**Issue by enlivenapp** [kenjis/codeigniter-composer-installer#36]
Hey Kenji,

TLDR;

`vendor/codeigniter4/framework/rewrite.php` and `vendor/codeigniter4/framework/serve` appear to be missing from the codeigniter download causing your installer to throw errors.

So, I used your installer just now and it looks like there's some changes in CI4 that's causing installation errors with your postInstall() script.

    > Kenjis\CodeIgniter\Installer::postInstall
    Script Kenjis\CodeIgniter\Installer::postInstall handling the post-create-project-cmd event terminated with an exception                                                                                                   
    [ErrorException]                                                                                   
     copy(vendor/codeigniter4/framework/rewrite.php): failed to open stream: No such file or directory  
                                                                                                     
I'm unsure why it's not throwing an exception for `serve` as well as that seems to not be in `vendor/codeigniter4/framework/` either.

**Comment by kenjis** [comment]
@enlivenapp Thank you for your reporting. I'm tring to fix the problem.

**Comment by enlivenapp** [comment]
weird.  I did just try it on my local machine and it worked fine.

Thanks for looking at it.
